src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,19 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        ColorSupplier firstSupplier = new ColorSupplier();
+        ColorSupplier secondSupplier = new ColorSupplier();
+        ColorSupplier thirdSupplier = new ColorSupplier();
+        Lottery firstBall = new Lottery();
+        Lottery secondBall = new Lottery();
+        Lottery thirdBall = new Lottery();
```

Comment:
1. Same as above
2. Lottery is not a ball, it's service providing balls so name ball for variable is not correct
```suggestion
        Lottery lottery = new Lottery();
```